src/common/subsys.h

```diff
@@ -83,6 +83,7 @@ SUBSYS(prioritycache, 1, 5)
 SUBSYS(test, 0, 5)
 SUBSYS(cephfs_mirror, 0, 5)
 SUBSYS(cephsqlite, 0, 5)
+SUBSYS(interrupt, 0, 5)
```

Comment:
How about `crimson_interrupt`? Simply `interrupt` might be pretty confusing for folks outside crimson.

Reply:
Yeah, I agree.